### PR TITLE
feat(ARC-2201): link translated pages with link alternate tag

### PR DESCRIPTION
**Pull request by bertyhell**
https://meemoo.atlassian.net/browse/ARC-2201

also replaced renderOgTags() helper with <SeoTags> component

image

image

image

questions page in english doesn't exist => no alternate tag is outputted
image

### CODE REVIEW

File: src/modules/account/views/MyFolders.tsx [hunk 1]
```diff
@@ -663,14 +663,18 @@ export const AccountMyFolders: FC<DefaultSeoInfo & AccountMyFolders> = ({ url, f
...
+					tText('pages/account/mijn-mappen/folder-slug/index___mijn-mappen') +
+					` | ${activeFolder?.name || folderSlug}`
+				}
+				description={tText(
 					'pages/account/mijn-mappen/folder-slug/index___mijn-mappen-meta-omschrijving'
-				),
-				url
-			)}
+				)}
+				imgUrl={undefined}
```
Comment (ian-emsens-shd) [inline src/modules/account/views/MyFolders.tsx:674]: `imgUrl` has a default `null` value, remove `imgUrl={undefined}`

File: src/modules/account/views/MyFolders.tsx [hunk 2]
```diff
@@ -663,14 +663,18 @@ export const AccountMyFolders: FC<DefaultSeoInfo & AccountMyFolders> = ({ url, f
...
+				translatedPages={[]}
```
Comment (ian-emsens-shd) [inline src/modules/account/views/MyFolders.tsx:675]: Same here, `translatedPages` defaults to `[]`

File: src/modules/search/ObjectDetailPage.tsx [hunk 3]
```diff
@@ -1596,7 +1596,14 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 	return (
 		<>
 			<VisitorLayout>
-				{renderOgTags(title, seoDescription, url, image)}
+				{' '}
```
Comment (ian-emsens-shd) [inline src/modules/search/ObjectDetailPage.tsx:1599]: Remove whitespace

File: src/modules/shared/components/SeoTags/SeoTags.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+			<meta property="og:title" content={resolvedTitle} />
+			{description && <meta property="og:description" content={description} />}
+			{imgUrl ? (
+				<meta property="og:image" content={imgUrl} />
+			) : (
+				<meta
+					property="og:image"
+					content={`${publicRuntimeConfig.CLIENT_URL}/images/og.jpg`}
+				/>
+			)}
```
Comment (ian-emsens-shd) [inline src/modules/shared/components/SeoTags/SeoTags.tsx:97]:
```suggestion
			<meta
				property="og:image"
				content={ imgUrl ? imgUrl : `${publicRuntimeConfig.CLIENT_URL}/images/og.jpg` }
			/>
```

File: src/pages/admin/alerts/index.tsx [hunk 5]
```diff
@@ -15,7 +16,7 @@ const AdminMaintenanceAlertsOverviewEnglish: FC<DefaultSeoInfo> = ({ url }) => {
 export async function getStaticProps(
 	context: GetServerSidePropsContext
 ): Promise<GetServerSidePropsResult<DefaultSeoInfo>> {
-	return getDefaultStaticProps(context);
+	return getDefaultStaticProps(context, undefined, ROUTES_BY_LOCALE.en.adminAlerts);
```
Comment (ian-emsens-shd) [inline src/pages/admin/alerts/index.tsx:19]: Consider swapping the order / switching to an `options` object so we can reduce these to e.g.
`getDefaultStaticProps(context, ROUTES_BY_LOCALE.en.adminAlerts)`

**Review by ian-emsens-shd — CHANGES_REQUESTED** [review]

File: src/modules/account/views/MyFolders.tsx (hunk 2 above)
Comment (bertyhell) [reply src/modules/account/views/MyFolders.tsx:675]: was already fixed in a later PR

File: src/modules/search/ObjectDetailPage.tsx (hunk 3 above)
Comment (bertyhell) [reply src/modules/search/ObjectDetailPage.tsx:1599]: was already fixed in a later PR

File: src/pages/admin/alerts/index.tsx (hunk 5 above)
Comment (bertyhell) [reply src/pages/admin/alerts/index.tsx:19]: fixed in commit on top of PR chain: https://github.com/viaacode/hetarchief-client/commit/f8ae3bf60cab6df8cf017cd5a24814c9568b5841

File: src/modules/shared/components/SeoTags/SeoTags.tsx (hunk 4 above)
Comment (bertyhell) [reply src/modules/shared/components/SeoTags/SeoTags.tsx:97]: fixed in commit on top of PR chain: https://github.com/viaacode/hetarchief-client/commit/f8ae3bf60cab6df8cf017cd5a24814c9568b5841

**Review by ian-emsens-shd — APPROVED** [review]

**Pull request merged**